homeassistant/components/tado/water_heater.py

```diff
@@ -212,6 +234,17 @@ def set_operation_mode(self, operation_mode):
 
         self._control_heater(hvac_mode=mode)
 
+    def set_timer(self, time_period, temperature):
+        """Set the timer on the entity, and temperature if supported."""
+        _LOGGER.info("Setting timer: time period %d temp %s", time_period, temperature)
```

Comment:
Please remove this or set it to debug level

Reply:
done